ytypes/util_schema.go

```diff
@@ -105,7 +105,10 @@ func validateListAttr(schema *yang.Entry, value interface{}) util.Errors {
 	if size < schema.ListAttr.MinElements {
 		errors = util.AppendErr(errors, fmt.Errorf("list %s contains fewer than min required elements: %d < %d", schema.Name, size, schema.ListAttr.MinElements))
 	}
-	if size > schema.ListAttr.MaxElements {
+	// 0 is an invalid value for MaxElements
+	// (https://tools.ietf.org/html/rfc7950#section-7.7.6).
+	// For useability it best represents the value "unbounded".
+	if schema.ListAttr.MaxElements != 0 && size > schema.ListAttr.MaxElements {
```

Comment:
Is there a reason that we shouldn't just error here?

Reply:
See above, I realized that this might create a useability issue.